### PR TITLE
test: add unit test for extension - part I

### PR DESCRIPTION
**Pull request by HuihuiWu-Microsoft**

### CODE REVIEW

File: packages/vscode-extension/test/index.integration.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ export async function run(): Promise<void> {
     reporterOptions: {
       reporterEnabled: "spec, mocha-junit-reporter",
       mochaJunitReporterReporterOptions: {
-        mochaFile: path.resolve(__dirname, "..", "..", "test-results.nofolder.xml"),
+        mochaFile: path.resolve(__dirname, "..", "..", "test-results.integration.xml"),
```
Comment (1openwindow) [inline packages/vscode-extension/test/index.integration.ts:18]: do we have integration test now?

File: packages/vscode-extension/src/controls/webviewPanel.ts [hunk 2]
```diff
@@ -69,7 +65,7 @@ export class WebviewPanel {
         // Enable javascript in the webview
         enableScripts: true,
         retainContextWhenHidden: true,
-        localResourceRoots: [vscode.Uri.file(path.join(this.extensionPath, "out"))],
+        localResourceRoots: [vscode.Uri.file(path.join(ext.context.extensionPath, "out"))],
```
Comment (1openwindow) [inline packages/vscode-extension/src/controls/webviewPanel.ts:68]: is any side effect on webpack. such as the path is different after webpack?

File: packages/vscode-extension/test/index.integration.ts (hunk 1 above)
Comment (HuihuiWu-Microsoft) [reply packages/vscode-extension/test/index.integration.ts:18]: Currently there's one. The original structure is <nofolder> = <unit test no folder> + <integration test no folder>, I separated according to unit test and integration test now. Should I clear the integration part in this PR?

**Review by 1openwindow — APPROVED** [review]

File: packages/vscode-extension/src/controls/webviewPanel.ts (hunk 2 above)
Comment (HuihuiWu-Microsoft) [reply packages/vscode-extension/src/controls/webviewPanel.ts:68]: Tested and works well

**Pull request merged**